contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -4,25 +4,21 @@
  * it will be copied to KeeperRegistry
  */
 
-pragma solidity ^0.7.0;
-pragma abicoder v2;
+pragma solidity 0.8.13;
 
-import "@openzeppelin/contracts-v0.7/utils/EnumerableSet.sol";
-import "../interfaces/AggregatorV3Interface.sol";
-import "../interfaces/LinkTokenInterface.sol";
-import "../interfaces/KeeperCompatibleInterface.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "../../v0.8/interfaces/AggregatorV3Interface.sol";
```

Comment:
move KeeperRegistryDev.sol to v0.8 folder instead?

Reply:
I was keeping it here just so that the git diff would be obvious. Let's move after we merge?